### PR TITLE
Removed unneeded no_settings_commands hardcoded list

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
Test failures seem unrelated.

File: django/core/management/base.py [hunk 1]
```diff
@@ -303,7 +303,11 @@ def run_from_argv(self, argv):
                 self.stderr.write('%s: %s' % (e.__class__.__name__, e))
             sys.exit(1)
         finally:
-            connections.close_all()
+            try:
+                connections.close_all()
+            except ImproperlyConfigured:
+                # Don't moan if connections aren't setup at this point
```
Comment (timgraham) [inline django/core/management/base.py]: Don't moan -> Ignore
Also, maybe explain under what circumstances that happens?

**Comment by timgraham** [comment]
Looks good except that the comment could be wrapped at 79.

**Comment by claudep** [comment]
Thanks, merged in fc92c6b.

**Pull request closed without merging**